multiqc/utils/search_patterns.yaml

```
@@ -469,6 +469,8 @@ somalier/pairs:
   fn: "*.pairs.tsv"
   contents: "hom_concordance"
   num_lines: 5
+pbmarkdup:
+  fn: "*.pbmarkdup.log"
```

See other comment above, better to search file contents if we don't have a standard filename from the tool.

```suggestion
  contents_re: "LIBRARY\s+READS\s+UNIQUE MOLECULES\s+DUPLICATE READS"
  num_lines: 5
```